scaffolds/scala/src/test/scala/euranova/AppleTest.scala

```diff
@@ -0,0 +1,17 @@
+package euranova
+
+import org.scalatest.FunSuite
+
+class MyTest extends FunSuite{
+
+	test("prop's val is OK") {
+	  val thing = new TheThing()
+	  assert("not OK value" != thing.theProp)
+	}
+  
+	test("create Thing") {
```

Comment:
It's definitely not mandatory but for the purpose of the code retreat, it might be cool to show the syntactic sugar of ScalaTest. You'll need to extend FlatSpec though.

    "TheThing" should "always return that it's ready" in {
        val thing = new TheThing()
        assert(thing.isReady === thing)
    }